manim/animation/creation.py

```diff
@@ -160,13 +161,11 @@ def _get_bounds(self, alpha: float) -> Tuple[int, float]:
         return (0, alpha)
 
 
+@deprecated(since="0.5.0", until="0.7.0", replacement="Create")
```

Comment:
There are few more things that are deprecated, see https://github.com/ManimCommunity/manim/projects/9 it would be nice to add that.

Reply:
I added all (as far as I could find them) deprecated classes, methods and parameters and based the `until` value on the time they were last changed plus two months.